src/services/user/useOwner.ts

```diff
@@ -45,6 +47,15 @@ export function useOwner({
     queryKey: ['owner', variables, provider, query],
     queryFn: ({ signal }) =>
       Api.graphql({ provider, query, variables, signal }).then((res) => {
+        const parsedData = DetailOwnerSchema.safeParse(res)
+        if (!parsedData.success) {
+          return Promise.reject({
+            status: 404,
+            data: {},
+            dev: 'useOwner - 404 failed to parse',
+          } satisfies NetworkErrorObject)
```

Comment:
Can we use the new `rejectNetworkError` helper function?

Reply:
fixed - was gonna make a sweep on this so you caught it before then - sorry for sloppy state then